src/Services/RedisModule.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Autofac;
+using Lykke.Job.OrderBooksCacheProvider.Core;
+using StackExchange.Redis;
+
+namespace Lykke.Job.OrderBooksCacheProvider.Services
+{
+    public class RedisModule : Module
+    {
+        private readonly CacheSettings _settings;
+
+        public RedisModule(CacheSettings settings)
+        {
+            _settings = settings;
+        }
+
+        protected override void Load(ContainerBuilder builder)
+        {
+            var redis = ConnectionMultiplexer.Connect(_settings.RedisConfiguration);
+
+            builder.RegisterInstance(redis).SingleInstance();
+            builder.Register(
+                c =>
+                    c.Resolve<ConnectionMultiplexer>()
+                        .GetServer(redis.GetEndPoints()[0]));
+
+            builder.Register(
+                c =>
+                    c.Resolve<ConnectionMultiplexer>()
+                        .GetDatabase());
```

Comment:
The documentation states that no need to store a database object
https://stackexchange.github.io/StackExchange.Redis/Basics.html#using-a-redis-database